flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -539,6 +524,79 @@ class FlxTilemapExt extends FlxTilemap
 	}
 	
 	/**
+	 * Sets the slope arrays, which define which tiles are treated as slopes of 22.5 degrees.
+	 * 
+	 * @param 	HighSlopes 	An array containing the numbers of the tiles to be treated as high slope tiles with a slope on the left.
+	 * @param 	LowSlopes	An array containing the numbers of the tiles to be treated as low slope tiles with a slope on the right.
+	 */
+	public function setSlopes22(?HighSlopes:Array<Int>, ?LowSlopes:Array<Int>) 
```

Comment:
Here's another thought. It looks kind of like a foot, right?

"Heel" and "Toe"